src/agentscope/studio/static/i18n/i18n_cn.json

```diff
@@ -0,0 +1,178 @@
+    "model-openai-chat-labelConfigName": "配置名称",
+    "model-openai-chat-labelModelName": "模型名称",
+    "model-openai-chat-Temperature": "温度",
+    "model-openai-chat-Seed": "种子",
+    "model-openai-chat-Advanced": "高级 &#x25BC;",
+    "model-openai-chat-advanced-box": "基准 URL",
+    "model-post-api-chat-readme": "Post API 聊天配置",
+    "model-post-api-chat-ConfigName": "配置名称",
+    "model-post-api-chat-ModelName": "模型名称",
+    "model-post-api-chat-Temperature": "温度",
+    "model-post-api-chat-Seed": "种子",
+    "model-post-api-chat-API": "API 地址",
+    "model-post-api-chat-ContentType": "Content-Type",
+    "model-post-api-chat-Authorization": "授权",
+    "model-post-api-chat-MessagesKey": "Messages Key",
+    "model-post-api-chat-MessagesKey-placeholder": "消息",
+    "model-post-api-dall-e-readme": "Post API Dall-E 配置",
+    "model-post-api-dall-e-ConfigName": "配置名称",
+    "model-post-api-dall-e-ModelName": "模型名称",
+    "model-post-api-dall-e-NumberofImages": "图片数量",
+    "model-post-api-dall-e-Temperature": "温度",
+    "model-post-api-dall-e-Seed": "种子",
+    "model-post-api-dall-e-API": "API 地址",
+    "model-post-api-dall-e-Authorization": "授权",
+    "model-post-api-dall-e-prompt-placeholder": "提示",
+    "model-wanx-readme":"Wanx 配置",
+    "model-wanx-ConfigName": "配置名称",
+    "model-wanx-ModelName": "模型名称",
+    "model-wanx-NumberofImages": "图片数量",
+    "model-wanx-size": "尺寸",
+    "model-wanx-Temperature": "温度",
+    "model-wanx-Seed": "种子",
+    "model-wanx-MessagesKey-placeholder": "提示",
+    "pipeline-forlooppipeline-readme":"用于实现 for 循环等控制流的模板管道",
+    "pipeline-forlooppipeline-MaxLoop": "最大循环次数",
+    "pipeline-forlooppipeline-BreakFunction": "中断函数",
+    "pipeline-ifelsepipeline-readme": "用 if-else 逻辑实现控制流的模板管道",
+    "pipeline-ifelsepipeline-Condition": "条件函数",
+    "pipeline-msghub-readme": "MsgHub 用于在一组代理之间共享信息",
+    "pipeline-msghub-AnnouncementName": "公告名称",
+    "pipeline-msghub-AnnouncementName-placeholder": "host",
+    "pipeline-msghub-AnnouncementContent": "公告内容",
+    "pipeline-msghub-AnnouncementContent-placeholder": "欢迎加入群聊!",
+    "pipeline-placeholder-readme":"不执行任何操作的占位符",
+    "pipeline-sequentialpipeline-readme": "实现顺序逻辑的模板管道（<b>从上到下</b）",
+    "pipeline-switchpipeline-readme": "用于使用 switch-case 逻辑实现控制流的模板管道",
+    "pipeline-switchpipeline-Condition": "条件函数",
+    "pipeline-switchpipeline-AddCase": "添加案例",
+    "pipeline-switchpipeline-RemoveCase": "删除案例",
+    "pipeline-whilelooppipeline-readme": "用于实现 while 循环等控制流的模板管道",
+    "pipeline-whilelooppipeline-Condition": "条件函数",
+    "service-bing-search-readme": "在 ReActAgent 中集成必应搜索服务，以增强代理功能",
+    "service-bing-search-Results": "结果编号",
+    "service-execute-python-readme": "在 ReActAgent中集成 Python 解释器，以增强代理功能",
+    "service-google-search-readme": "将 Google 搜索服务集成到 ReActAgent 中以增强代理功能",
+    "service-read-text-readme": "将阅读文本服务集成到 ReActAgent 中以增强代理功能",
+    "service-write-text-readme": "将写入文本服务集成到 ReActAgent 中提升代理能力",
```

Comment:
There are more services and tools in workstation, please add translations for them.